mixing/errors.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package mixing
+
+import (
+	"errors"
+	"fmt"
+)
+
+var (
+	ErrInvalidPROrder = errors.New("invalid pair request order")
```

Comment:
They could just be unexported if you prefer that.  It just ends up getting wrapped in a mixpool `RuleError` anyways, which does follow the practices afaict.  Right now it is only declared as a global variable because a test is checking for this particular condition.

Reply:
unexported these.